src/views/virtualmachines/utils/virtualMachineRowFilter.ts

```diff
@@ -1,10 +1,15 @@
+import { TFunction } from 'i18next';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { OTHER } from '@kubevirt-utils/components/DiskModal/DiskFormFields/utils/constants';
```

Comment:
not sure we want to use OTHER from disks :-)

Reply:
changed to using const other = t('Other') instead